skimage/measure/_regionprops.py

```diff
@@ -16,36 +16,52 @@
 __all__ = ['regionprops', 'euler_number', 'perimeter', 'perimeter_crofton']
 
 
+# All values in this PROPS dict correspond to current scikit-image property
+# names. The keys in this PROPS dict correspond to older names used in prior
+# releases. For backwards compatibility, these older names will continue to
+# work, but will not be documented.
 PROPS = {
     'Area': 'area',
     'BoundingBox': 'bbox',
-    'BoundingBoxArea': 'bbox_area',
+    'BoundingBoxArea': 'area_bbox',
+    'bbox_area': 'area_bbox',
     'CentralMoments': 'moments_central',
     'Centroid': 'centroid',
-    'ConvexArea': 'convex_area',
+    'ConvexArea': 'area_convex',
+    'convex_area': 'area_convex',
     # 'ConvexHull',
-    'ConvexImage': 'convex_image',
+    'ConvexImage': 'image_convex',
+    'convex_image': 'image_convex',
     'Coordinates': 'coords',
     'Eccentricity': 'eccentricity',
     'EquivDiameter': 'equivalent_diameter',
     'EulerNumber': 'euler_number',
     'Extent': 'extent',
     # 'Extrema',
     'FeretDiameterMax': 'feret_diameter_max',
```

Comment:
The legacy name of `FeretDiameterMax` was `FeretDiameter`. `FeretDiameterMax` already introduced in https://github.com/scikit-image/scikit-image/pull/4820 and I think it breaks backwards compatibility. Now would be a good time to fix it.

Reply:
Thanks, I have now made this change